### PR TITLE
Add jubakit

### PR DESCRIPTION
**Pull request by shiodat**

### CODE REVIEW

File: hands-on-5th/jubakit/jubakit.ipynb [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jubakit.wrapper.classifier import LinearClassifier\n",
+    "\n",
+    "clf = LinearClassifier(method=\"AROW\", regularization_weight=0.1)\n",
```
Comment (imaimai1125) [inline hands-on-5th/jubakit/jubakit.ipynb]: defalutでembedded Jubatusを呼ぶようになっているので、embedded=Falseを追加しましょう
あるいはdockerfileにcythonとembedded Jubatusの追加を
(pip等のverのすり合わせもしたいですね)

File: hands-on-5th/jubakit/jubakit.ipynb [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+   "source": [
+    "from jubakit.wrapper.classifier import LinearClassifier\n",
+    "from sklearn.model_selection import GridSearchCV\n",
+    "\n",
+    "param_grid = {\n",
+    "    \"method\": [\"AROW\", \"CW\"],\n",
+    "    \"regularization_weight\": [0.01, 0.1, 1.0, 10, 100]\n",
+    "}\n",
+    "\n",
+    "clf = LinearClassifier()\n",
```
Comment (imaimai1125) [inline hands-on-5th/jubakit/jubakit.ipynb]: embedde=False

File: hands-on-5th/jubakit/jubakit.ipynb [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    "searcher = GridSearchCV(clf, cv=4, param_grid=param_grid, verbose=2)\n",
```
Comment (imaimai1125) [inline hands-on-5th/jubakit/jubakit.ipynb]: このあたり何をしているかの説明は、スライドでかんたんにお願いします。

File: hands-on-5th/jubakit/jubakit.ipynb [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from operator import itemgetter\n",
+    "means = searcher.cv_results_['mean_test_score']\n",
+    "params = searcher.cv_results_['params']\n",
+    "grid_scores = sorted([(mean, param) for mean, param in zip(means, params)], key=itemgetter(0), reverse=True)\n",
```
Comment (imaimai1125) [inline hands-on-5th/jubakit/jubakit.ipynb]: Scoreが何なのか明記(Accuracy?)

**Review by imaimai1125 — CHANGES_REQUESTED** [review]
いくつかコメントしておきました。

**Comment by shiodat** [comment]
@imaimai1125 I add some descriptions and fix code.

**Review by imaimai1125 — APPROVED** [review]

**Comment by imaimai1125** [comment]
LGTM!
